services/web/server/src/simcore_service_webserver/projects/exceptions.py

```diff
@@ -25,6 +25,10 @@ class ProjectInvalidUsageError(BaseProjectError):
     ...
 
 
+class ProjectOwnerNotFoundInTheProjectAccessRightsError(BaseProjectError):
+    msg_template = "Project owner gid with required permissions was not found in the project access rights"

```

Comment:
Maybe add the `gid` and the access rights requested?

Reply:
I didn't put the gid intentionally there, as I do not want to expose the owner gid value if somebody would try to access this endpoint without having proper rights.